builder/steps/gen-dockerfile/contents/src/generate_files.ts

```diff
@@ -77,14 +77,12 @@ export async function generateFiles(
 
   // Generate the Dockerfile and remove any empty lines
   const dockerfileTemplate = await dataDirReader.read('Dockerfile.txt');
-  const dockerfile = renderTemplate(dockerfileTemplate, {
-    baseImage: baseImage,
-    tool: config.useYarn ? 'yarn' : 'npm',
-    config: config
-  });
+  const dockerfile = renderTemplate(
+      dockerfileTemplate,
+      {baseImage, tool: config.useYarn ? 'yarn' : 'npm', config});
 
   const dockerignoreTemplate = await dataDirReader.read('dockerignore');
-  const dockerignore = renderTemplate(dockerignoreTemplate, {config: config});
+  const dockerignore = renderTemplate(dockerignoreTemplate, {config});
```

Comment:
Is removing `config` expected?

Reply:
`{config}` is syntactic sugar for `{config: config}` and so it is not unexpected.